pages/api/trips/[id].js

```diff
@@ -22,5 +25,7 @@ export default async function handler(request, response) {
       response.status(500).json({ message: "Error deleting service" });
     }
     return;
+  } else {
+    return;
```

Comment:
My suggestion is to add  a `405 status` code for unsupported methods. 

```suggestion
  } else {
    return response.status(405).json({ message: "Method not allowed" });
```